### ISSUE TITLE
Enable or Disable buttons depending on modality

### ISSUE THREAD

**Issue by SathiWork** [Sriram-Ananth/EDGEPACS#37]
minor enhancement: Can we disable some buttons like probe tool and reference lines depending on modality (eg. Ultrasound).

**Comment by Sriram-Ananth** [comment]
Yes we need to. I will add toolbar management to the modality settings. Maybe v2

**Comment by SathiWork** [comment]
Yes Yes.
